core/xml.js

```diff
@@ -383,9 +383,13 @@ Blockly.Xml.textToDom = function(text) {
  */
 Blockly.Xml.clearWorkspaceAndLoadFromXml = function(xml, workspace) {
   workspace.setResizesEnabled(false);
-  workspace.clear();
-  var blockIds = Blockly.Xml.domToWorkspace(xml, workspace);
-  workspace.setResizesEnabled(true);
+  try {
+    workspace.clear();
+    var blockIds = Blockly.Xml.domToWorkspace(xml, workspace);
```

Comment:
Travis is complaining because this change the return type:

https://travis-ci.org/github/google/blockly/jobs/694866312#L9283